src/components/Navigator.jsx

```diff
@@ -0,0 +1,17 @@
+const Navigator = () => {
+  return (
+    <ul class="nav-list">
+      <li>
+        <button className="navigator-item">Cooking</button>
+      </li>
+      <li>
+        <button className="navigator-item">Coding</button>
+      </li>
+      <li>
+        <button className="navigator-item">Football</button>
+      </li>
+    </ul>
```

Comment:
Semantically you shouldn't use buttons for navigation. I'm aware these don't work yet. Screen readers read buttons for a function such as submit. These should be Links. You can certainly style links to look like buttons but the core code shouldn't be a button.